src/physics/nonlinear_solid.hpp

```diff
@@ -278,12 +278,12 @@ class NonlinearSolid : public BasePhysics {
   /**
    * @brief Nonlinear system solver instance
    */
-  EquationSolver nonlin_solver_;
+  mfem_extensions::EquationSolver nonlin_solver_;
```

Comment:
Same question. Do we want to mark things which could be upstreamed to mfem or only derived objects?

Reply:
Hmm, I'm not sure what would qualify as potentially upstreamable to MFEM.  My thinking was that part of what the `mfem_extensions` namespace would indicate is that things like function names follow the MFEM instead of serac convention.